api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -463,6 +465,22 @@ public void validateTermsOfService(@Nonnull DbUser dbUser) {
     validateTermsOfService(tosVersion);
   }
 
+  @Override
+  public boolean validateTerraTermsOfService(@Nonnull DbUser dbUser) {
+    boolean termsOfServiceAccepted = false;
+    try {
+      termsOfServiceAccepted = fireCloudService.getUserTermsOfServiceStatus();
+    } catch (ApiException e) {
```

Comment:
should we catch this exception, or simply let the underlying exception propagate?

Reply:
Out of curiosity, where does log.log get stored to? Does it go into GCP somewhere?

Reply:
I wanted to log the message with userId, was not sure if the underlying exception will do the same
@evrii  yes it goes to google cloud console logger